src/orders/components/OrderPaymentSummaryCard/components/PaymentsSummary/types.ts

```diff
@@ -0,0 +1,8 @@
+export enum PaymentState {
+  NO_DATA,
+  AMOUNTS_MISMATCH,
+  PARTIAL_CAPTURE,
+  PARTIAL_AUTHORIZED,
+  FULLY_SETTLED,
+  IS_PENDING,
+}
```

Comment:
**question:** shouldn't this come from the api and be in generated types?

Reply:
Nope, this doesn't have to do anything with the API (except with similar names I guess). It's a description for transaction amount states (e.g. **amounts mismatch** - it doesn't exist in the API, but we need to know that the amounts are different, because in such case we want to display them; other example: **fully settled** - it exists in the API, but in our case it just means we don't want to display the amounts not that the order is paid)